modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase

```diff
@@ -64,8 +64,49 @@ definition {
 	test CheckFieldsetSummaryLocalization {
 		property portal.acceptance = "false";
 
-		// TODO LPS-123539 CheckFieldsetSummaryLocalization pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Web Content");
 
+		NavItem.gotoStructures();
+
+		WebContentStructures.addCP(
+			structureDescription = "Structure Description Test",
+			structureName = "Structure FieldSet Test");
+
+		DataEngine.addField(
+			fieldFieldLabel = "Text Test",
+			fieldName = "Text");
+
+		WebContentStructures.saveCP(structureName = "Structure Name Test");
+
+		WebContentStructures.addCP(
+			structureDescription = "Structure Description Test",
+			structureName = "Structure Name Test");
+
+		DataEngine.addFieldsets(
+			fieldsetLabel = "Structure FieldSet Test",
+			fieldsetName = "Structure FieldSet Test");
+
+		WebContentStructures.saveCP(structureName = "Structure Name Test");
+
+		NavItem.click(navItem = "Web Content");
+
+		WebContentNavigator.gotoAddWithStructureCP(structureName = "Structure Name Test");
+
+		PortletEntry.inputTitle(title = "WC Title Test");
+
+		PortletEntry.inputText(
+			fieldLabel = "Text",
+			text = "Text test");
+
+		PortletEntry.changeLocaleSummary(locale = "pt_BR");
+
+		Pause(locator1 = "5000");
+
+		PortletEntry.changeLocale(locale = "es_ES");
+
```

Comment:
Also needs an assert here to check that the language changed.

Reply:
Line 113 checks to see if the locale has switched to ES